extensions/positron-python/package.json

```diff
@@ -681,6 +686,16 @@
                     "scope": "resource",
                     "type": "string"
                 },
+                "python.useEnvironmentsExtension": {
```

Comment:
do we want to expose this, even though we do not bundle the environments extension?